src/main/java/com/revature/chronicle/security/MasonTestController/TestController.java

```diff
@@ -0,0 +1,19 @@
+package com.revature.chronicle.security.MasonTestController;
+
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("mason-test")
+public class TestController {
+
+    @GetMapping(produces= MediaType.APPLICATION_JSON_VALUE)
+    @ResponseBody
+    public String testMason() {
+        return "User is logged in";
```

Comment:
Let's just keep one controller like this.

Reply:
I changed it to be like that but removed all controllers, if someone wants the test controller, they can @justinkroh22. The controller will be added to gitignore